internal/cmd/policy/reorderrules/reorderrules.go

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package reorderrules
+
+import (
+	"github.com/gittuf/gittuf/internal/cmd/common"
+	"github.com/gittuf/gittuf/internal/cmd/policy/persistent"
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	p          *persistent.Options
+	policyName string
+	ruleNames  []string
+}
+
+func (o *options) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&o.policyName,
+		"policy-name",
+		"targets",
+		"name of policy file to reorder rules in",
+	)
+
+	cmd.Flags().StringSliceVar(
+		&o.ruleNames,
+		"rule-order",
+		[]string{},
+		"a space-separated list of rule names, in the new order that they should appear in, from first to last",
```

Comment:
does the flag not need to be repeated? as in `--rule-name <> --rule-name <> ...`

Reply:
It can work both ways, either repeated or not repeated